internal/logger/component.go

```diff
@@ -125,6 +125,7 @@ type Command struct {
 	// TODO(GODRIVER-2824): change the DriverConnectionID type to int64.
 	DriverConnectionID uint64              // Driver's ID for the connection
 	Name               string              // Command name
+	DatabaseName       string              // Database name
```

Comment:
I'm not completely sure, but I think it would be ideal to only keep "common fields" as defined in the driver [specifications](https://github.com/qingyang-hu/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#common-fields) in these structs. Other fields should be added to the key/values inline, like this: https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/operation.go#L1795

Reply:
I've updated the spec at: https://github.com/mongodb/specifications/commit/6b267ddc7cca303a722d2893106fad4810709f06#diff-81d41cc6fd279ee033885bc989eea1823c252f4b3e0c00a8605e03f1290ef864R402

Reply:
The common logging fields are defined here: https://github.com/qingyang-hu/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#common-fields

Reply:
I see my mistake, thank you for clarifying offline!